src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -14,240 +14,227 @@
-            "Premium Videos - Sktest123 Wiki - Wikia",
-            "Premium Videos - Wikia"
+            "Category:Premium Videos - Sktest123 Wiki - Wikia",
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "Message_Wall:Sktest",
             "Message Wall:Sktest - Sktest123 Wiki - Wikia",
-            "Message Wall:Sktest - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "Thread:2160",
             "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia",
-            "Welcome to Sktest123 Wiki! - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "Special:Maps",
-            "Maps - Wikia",
-            "Maps - Wikia"
+            "Maps - Sktest123 Wiki - Wikia",
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "Template:Welcome",
             "Template:Welcome - Sktest123 Wiki - Wikia",
-            "Template:Welcome - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "MediaWiki:Common.css",
             "MediaWiki:Common.css - Sktest123 Wiki - Wikia",
-            "MediaWiki:Common.css - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "MediaWiki:Edit",
             "MediaWiki:Edit - Sktest123 Wiki - Wikia",
-            "MediaWiki:Edit - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "User:Sktest",
             "User:Sktest - Sktest123 Wiki - Wikia",
-            "User:Sktest - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "File:THE_HOBBIT_Trailer_-_2012_Movie_-_Official_HD",
             "Video - THE HOBBIT Trailer - 2012 Movie - Official HD - Sktest123 Wiki - Wikia",
-            "Video - THE HOBBIT Trailer - 2012 Movie - Official HD - Sktest123 Wiki - Wikia"
         },
         {
             TEST_WIKI_ORIGINAL_TITLE,
             "File:Giant_prominence_on_the_sun_erupted.jpg",
             "Image - Giant prominence on the sun erupted.jpg - Sktest123 Wiki - Wikia",
-            "Image - Giant prominence on the sun erupted.jpg - Sktest123 Wiki - Wikia"
         },
-        // custom title
+        {
+            TEST_WIKI_ORIGINAL_TITLE,
+            "Special:NonExistingSpecialPage",
+            "Error - Sktest123 Wiki - Wikia",
+        },
+        // Em-dash instead of regular dash for some languages
+        {
+            TEST_WIKI_ORIGINAL_TITLE_WITH_EM_DASH,
+            "Ulica_Kazimierza_Wielkiego",
+            "Ulica Kazimierza Wielkiego – Poznańska Wiki – Wikia",
+        },
+        // Custom title
         {
             TEST_WIKI_CUSTOM_TITLE,
             "WikiDex",
             "WikiDex, la enciclopedia Pokémon - Wikia",
-            "WikiDex - WikiDex - Wikia"
         },
+        // TODO: SEO-77 Fix the title of the main pages for Mercury
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)